prog/types.go

```diff
@@ -529,6 +530,9 @@ func (t *BufferType) DefaultArg(dir Dir) Arg {
 }
 
 func (t *BufferType) isDefaultArg(arg Arg) bool {
+	if t.Kind == BufferGlob {
+		return true
```

Comment:
This does not look right.
If the arg has default value, it won't be serialized. But then DefaultArg function is expected to restore the value. If we have, say, 10 files, there is no way DefaultArg will be able to guess which one we had before serialization.
I would say we should never consider it to have default value.

Reply:
Did you forget to push? This still holds.